DynatraceExtensions.cs

```diff
@@ -49,8 +50,8 @@ public static class DynatraceExtensions
                 Environment.GetEnvironmentVariable("ASPNET_ENVIRONMENT");
 
             return sinkConfiguration.Http(ingestUrl,
-                batchPostingLimit: batchPostingLimit ?? 50,
-                queueLimit: queueLimit ?? 100,
+                batchSizeLimitBytes: batchSizeLimitBytes,
+                queueLimitBytes: queueLimitBytes,
```

Comment:
This change removes the default value, many times those who use the LIB do not know this and when updating it can cause the application to crash.